connector_jira/models/project_task/importer.py

```diff
@@ -108,6 +111,35 @@ def parent(self, record):
     def backend_id(self, record):
         return {'backend_id': self.backend_record.id}
 
+    @mapping
+    def status(self, record):
+        status = record['fields'].get('status', {})
+        status_name = status.get('name')
+        if not status_name:
+            return {'stage_id': False}
+        project_binder = self.binder_for('jira.project.project')
+        project_id = project_binder.unwrap_binding(
+            self.options.project_binding
+        )
+        stage_id = self.env['project.task.type'].search(
+            [
+                ('name', '=', status_name),
+                ('project_ids', '=', project_id.id),
+            ],
+            limit=1,
+        )
+        return {'stage_id': stage_id.id if stage_id else False}
+
+    @mapping
+    def time_estimate(self, record):
+        original_estimate = record['fields'].get('timeoriginalestimate')
+        if not original_estimate:
+            return {'planned_hours': False}
+        planned_hours = (
+            timedelta(seconds=int(original_estimate)) / timedelta(hours=1)
```

Comment:
Is there a reason for needing `timedelta` here? `float(original_estimate) / 3600.` is equal.

Reply:
No specific reason at all, just other way of expressing the same